app/src/androidTest/java/com/github/sdp/ratemyepfl/activity/CourseReviewActivityTest.kt

```diff
@@ -34,10 +39,15 @@ class CourseReviewActivityTest {
         scenario.close()
     }
 
+
     @Test
     fun isIdVisibleOnActivityLaunch() {
+        val fakeCourseId = FakeCourseRepository.COURSE_BY_ID.id
+        Thread.sleep(1000)

```

Comment:
What is the utility of this wait?

Reply:
Same question :(

Reply:
It's useless :) fun fact this was not working and I wanted to see the layout during the test. I forgot to remove it.

Reply:
I use the same trick for tests :)